dotCMS/src/main/java/com/dotmarketing/portlets/languagesmanager/business/LanguageAPI.java

```diff
@@ -241,6 +241,13 @@
      */
     public Language getFallbackLanguage(final String languageCode);
 
+	/**
+	 * Finds the first language with the language code
+	 * @param languageCode String
+	 * @return Optional language
+	 */
+	public Optional<Language> getFindFirstLanguageByCode(final String languageCode);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary modifier 'public' on method 'getFindFirstLanguageByCode': the method is declared in an interface type](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4324168)